src/Main.ts

```diff
@@ -1,23 +1,19 @@
-"use strict";
-
 /* -----------------------------------------------
 /* Author : Matteo Bruni - https://www.matteobruni.it
 /* MIT license: https://opensource.org/licenses/MIT
 /* Demo / Generator : https://particles.matteobruni.it/
 /* GitHub : https://www.github.com/matteobruni/tsparticles
 /* How to use? : Check the GitHub README
-/* v1.10.2
+/* v1.11.0
 /* ----------------------------------------------- */
-import {Container} from "./Classes/Container";
-import {Loader} from "./Classes/Loader";
-import {IParticlesJs} from "./Interfaces/IParticlesJs";
-import {ParticlesJS} from "./support";
-import {IOptions} from "./Interfaces/Options/IOptions";
-import {RecursivePartial} from "./Types/RecursivePartial";
+import type { Container } from "./Classes/Container";
+import { Loader } from "./Classes/Loader";
+import type { IOptions } from "./Interfaces/Options/IOptions";
+import type { RecursivePartial } from "./Types/RecursivePartial";
 
 declare global {
     interface Window {
-        particlesJS: IParticlesJs;
+        particlesJS: any;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)